src/Murder/Assets/GameProfile.cs

```diff
@@ -150,6 +150,7 @@ public float Aspect
         public readonly bool IsVSyncEnabled = false;
         public readonly bool ShowUiDebug = true;
         public readonly float PushAwayInterval = 0.05f;
+        public readonly int DefaultGridCellSize = 24;
```

Comment:
I think a better solution for this is adding this to `WorldAsset` instead (the editor for this is in the ["Settings" tab](https://github.com/isadorasophia/murder/blob/71f3f04d0e8d7b1106addb0b7096e2335d093892/src/Murder.Editor/CustomEditors/WorldAssetEditor.cs#L446)). So when the game [switches scenes](https://github.com/isadorasophia/murder/blob/71f3f04d0e8d7b1106addb0b7096e2335d093892/src/Murder/Core/Scenes/SceneLoader.cs#L39), it can track the Grid property through there. Probably we can wrap that around a `protected` method inside Game that handles a clean switch before calling `SceneLoader.SwitchScene`.

It gets tricky because we probably need to pass that down to the stage editor as well... @saint11 if the prefab editor ever asks for a grid (e.g. calculating the size), what should we do?

Reply:
For prefabs it should use the "default" one specified on Profile. So I think this is correct!
In the future we will want to be able to override this value with one specified in each WorldAsset

Reply:
Hmmm got it. So we can leave the `WorldAsset` stuff for the future, and just keep the `GameProfile` one for now.